### DISCUSSION TITLE
Mushroom Overhaul/Splitting

### DISCUSSION THREAD

**Discussion by Tefnut** [cataclysmbn/Cataclysm-BN#3209] [General]
Mushrooms as a whole are dragged down by all being classified as one item. It causes batch crafting issues. The only upside they provide is the identification mechanics, and the chance of accidentally eating poison, which is needless bloat.

My recommendation to fix this issue is simple; make the three forms of mushroom separate items, and make it work similar to mutant/human meat in recipes. Poison ones, of course, probably not being usable in cooking until theres a reason to, say, give NPCs poisoned food); but hallucinogenic mushrooms pass down their hallucinogenic nature (And possibly a small joy buff since you're intentionally putting hallucinogens in your food?), while renaming food items only if human or mutant meat isnt used in them.

Naming is the hardest part, so I will do that part for hallucinogenic mushrooms, explaining any references. You may or may not use these ideas. I'm going for a sort of hippie beatnik theming for most of them.

Pickled Veggie -> Pickled Psilocybin (Scientific name for the chemical that makes you hallucinate)
Woods Soup -> Woodstock Soup (A reference to the music festival infamous for people using mind altering substances)
Fish Soup -> Phish Soup (A reference to a rock band from the 80s who had psychadelic influences in some of their music)
Pemmican -> Psychadelic Pemmican
Spaghetti Al Pesto -> Spaghetti al Pinko (A derogatory term against left leaning, especially commie-leaning, people, was used against hippies)
Broth -> Beatnik Broth
Mushroom Soup -> Magic Mushroom Soup
Sausage Gravy -> Grateful Sausage Gravy (Grateful Dead)
Supreme Pizza -> Magical Mystery Pizza (The Beatles Magical Mystery Tour album, which was made under the influence)
Vegetable Pizza -> Veg-Out Pizza (Vegging out, something people tend to do when theyre high)
Deluxe Sandwich -> Drugged Deluxe Sandwich
Curry -> Chic Curry (I'm scraping the bottom of the barrel for some of these)
Deluxe Beans and Rice -> Drugged Deluxe Beans and Rice (The sandwich and this are very much "if you can't think of anything better"
Mushroom Omelette -> Sacred Mushroom Omelette (Sacred mushroom is another slang used for magic mushrooms).

**Comment by Charlotte-br560** [comment]
Great suggestions! Differentiating mushroom types in crafting could indeed add depth to the game. Your proposed names provide a playful and thematic touch, enhancing the overall experience. The implementation of these ideas would not only address the batch crafting issues but also contribute to the immersive and creative aspects of gameplay. Consider submitting a pull request with these changes, as it aligns well with the game's spirit. Keep up the fantastic work!
